acid/app.py

```diff
@@ -39,5 +40,6 @@
 
 @app.context_processor
 def template_context():
-    return {'pipeline_names': app.config['zuul']['pipelines'],
+    return {'pipeline_names': pipe_intersect(app.config['zuul']['pipelines'],
```

Comment:
Good idea, wrong execution.
pipe_intersect is a utility function, true, but imo is not in any way related to any feature - like status. Its just a logical mechanic.
Might want to put it somewhere else, not in features, like utils.py or somewhere more relevant.